make/common.mk

```diff
@@ -29,6 +29,12 @@ include $(BASE_MAKE)/series-folder-name.mk
 include $(BASE_MAKE)/$(SERIES_FOLDER)/common.mk
 MAPPED_DEVICE ?= $(DEVICE)
 
+# Default flashing tool is st-flash
+FLASHING_TOOL = st-flash
+
+# Select serial port in case of using serial interface
+SERIAL_PORT = /dev/ttyUSB0
```

Comment:
Does it make sense to set this variable as a default? I don't know enough about USB device naming policies for both Linux and MacOS to make a call on this yet. The name assignments of serial ports in Windows feels kinda random.

This file could be set up in a way which makes the `SERIAL_PORT` variable required if the tool `stm32flash` is chosen.